content/cloud-computing/concepts/cloud-services/cloud-services.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Cloud Services'
+Description: 'There are many different cloud-computing services, but the big three are Microsoft Azure, Amazon Web Services (AWS) and Google Cloud Platform (GCP).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are many different cloud-computing services, but the big three are [Amazon Web Services (AWS)](https://aws.amazon.com/), [Microsoft Azure](https://azure.microsoft.com/), and [Google Cloud Platform (GCP)](https://cloud.google.com/). All three offer services to cover the most significant cloud-computing needs:
+
+- [Infrastructure as a service (IaaS)](https://www.codecademy.com/resources/docs/cloud-computing/iaas)
+- [Platform as a service (PaaS)](https://www.codecademy.com/resources/docs/cloud-computing/paas)
+- [Function as a service (FaaS)](https://www.codecademy.com/resources/docs/cloud-computing/faas)
+- [Software as a service (SaaS)](https://www.codecademy.com/resources/docs/cloud-computing/saas)
+
+## AWS
+
+AWS is the largest cloud computing platform, and the most mature, introducing its first public cloud services in 2006. It serves millions of users, from individuals to large enterprises, to governments. Offering more than 200 services to its customers, AWS has the largest catalog of services of the big three. As of 2021 it has 25 geographic regions and 218+ edge locations.
+
+The advantages of AWS include:
+
+- The widest array of services; from networking to quantum computing.
+- The most mature, reliable and secure platform.
+- The most compute capability.
+
+Disadvantages include:
+
+- Users must purchase Dev/Enterprise support.
+- The number of services and options can be overwhelming.
+- A complex and hard-to-understand pricing structure.
+- There are limited options to run a [hybrid cloud](https://www.codecademy.com/resources/docs/cloud-computing/types-of-cloud) solution.
+
+## Azure
+
+Azure is the second largest cloud platform, introduced my Microsoft in 2008. It is also currently among the fastest growing cloud platforms. It also has more than 200 products and services, a wide array of which are tailored to Microsoft-centric enterprises, providing an easier transition for many enterprises to a cloud or hybrid-cloud environment. In addition to Microsoft products, it also supports [open-source](https://www.codecademy.com/resources/docs/open-source) languages, technologies and platforms. As of 2021 it has 60+ geographic regions with 116 edge locations. It has the broadest geographic reach of the three platforms. It also tends to have the lowest on-demand pricing for enterprise users already using Microsoft services such as Windows, Active Directory and SQL Server.
```

Comment:
I don't think its very clear what "Microsoft centered enterprises" means. Also I would consider breaking out these larger paragraphs both here and in AWS and GCP into more lists or subheaders, this is a lot of dense information.